interface/src/app/utils.js

```diff
@@ -17,4 +17,24 @@ angular.module('adage.utils', [])
 
 // Number of digits in activity value shown on frontend:
 .constant('ActivityDigits', 5)
+
+// Service of global machine learning model, which is shared by multiple
+// modules.
```

Comment:
@mhuyck : Practically your approach will work, but I would prefer adding the dependency of this new module in the module that uses this `MlModelTracker` directly, otherwise when you do unit test, you will have to always remember to include `adage.utils` module, even though you may not really need it in the test.